src/demo/index.php

```diff
@@ -26,7 +26,9 @@ function gtag() { dataLayer.push(arguments); }
     <script async defer src="https://buttons.github.io/buttons.js"></script>
 </head>
 
-<body <?php echo $_COOKIE["darkmode"] == "on" ? 'data-theme="dark"' : ""; ?>>
+<body <?php if (!empty($_COOKIE)) {
+        echo $_COOKIE["darkmode"] == "on" ? 'data-theme="dark"' : "";
+} ?>>
```

Comment:
isset should be used to check, and can keep it on 1 line

```suggestion
<?php echo (isset($_COOKIE["darkmode"]) && $_COOKIE["darkmode"] == "on") ? 'data-theme="dark"' : ""; ?>
```

Reply:
Do you want me to do this?

Reply:
I was going to include isset(). Removing the is not empty if-condition removes the fix so I have made a new commit with both.